qa/config/jdbc_profiles/default/JDBCProfiles

```diff
@@ -164,20 +171,20 @@ DB1_THIN_JNDI_NumberOfDrivers=2
 DB1_THIN_JNDI_Driver0=oracle.jdbc.driver.OracleDriver
 DB1_THIN_JNDI_Driver1=com.arjuna.ats.jdbc.TransactionalDriver
 DB1_THIN_JNDI_Binding=oracle_thin1
-DB1_THIN_JNDI_DatabaseURL=jdbc:arjuna:oracle_thin1
+DB1_THIN_JNDI_DatabaseURL=jdbc:arjuna:oracle_thin1:@//narayanaci1.eng.hst.ams2.redhat.com:1521/XEPDB1
```

Comment:
@jmfinelli please can you clarify why you have needed to add this here? I am wondering about backwards compatibility if you have had to manually add the details here why a change to DB1_THIN_JNDI_DatabaseName was not enough

Reply:
This connection string works fine using the oracle driver but the TransactionalDriver has a different opinion :-)
Thus, I am still not sure about this change. I pushed it only to trigger the testing axis (but the test failed). I am now working locally (creating a reproducer) to understand why the TransactionalDriver does not like any one of `jdbc:arjuna:oracle_thin` and `jdbc:arjuna:oracle_thin1:@//narayanaci1.eng.hst.ams2.redhat.com:1521/XEPDB1` when I want to connect to a pluggable database. The real question though is "why has oracle changed its behaviour now?". To answer that, I will also try to rebuild oracle from scratch (starting from building the image) and see if we can just scrap this PR.

Reply:
Thanks @jmfinelli - I have dismissed my previous review so I can understand the backwards compatbility aspect better. I have a feeling that the configuration of the driver is built up from various parts of the config file.

Reply:
@jmfinelli it seems from the surefire report xml file for ActivationRecordUnitTest that something is wrong with the username/password.